schemas/eq_submission_schema.json

```diff
@@ -197,22 +197,27 @@
     "case_type": {
       "$id": "#/properties/case_type",
       "type": "string",
+      "items": {
+        "type": "array",
```

Comment:
Runner does  not accept an array at the moment?

I think this should be type string?

Reply:
I've made requested changes, thanks.